main.go

```diff
@@ -263,7 +262,11 @@ func main() {
 		}()
 	}
 
-	ctrl.SetLogger(klogr.New())
+	lggr := klogr.New()
+	if callDepthLogger, ok := lggr.(logr.CallDepthLogger); ok {
+		lggr = callDepthLogger.WithCallDepth(2)
```

Comment:
why are we doing this here and why is it 2 and not 3 like in composite_logger.go?

Reply:
This was not effective and has been removed.